docs/UserGuide.md

```diff
@@ -207,6 +223,7 @@ Format: `exit`
  | **Delete** | `delete INDEX`<br> e.g., `delete 3`|
 | **Edit**   | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [i/INSURANCE_PACKAGE] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`|
  | **Find**   | `find FIELD KEYWORD [MORE_KEYWORDS]`<br> e.g., `find n/James Jake`|
+ | **Clip**    | `clip n/NAME`<br> e.g., `find n/John Doe`|
```

Comment:
Did you mean "e.g., `clip n/John Doe`"?

Reply:
Similar to how after a list or find, `delete INDEX` deletes the details at that specified INDEX, perhaps `clip INDEX` can do something similar?

Reply:
That definitely sounds like a good enhancement for the clip command! Great idea!